packages/uui-pagination/lib/uui-pagination.element.ts

```diff
@@ -214,11 +245,9 @@ export class UUIPaginationElement extends LitElement {
       look="outline"
       class="nav"
       role="listitem"
-      label="Go to first page"
+      label=${this.firstLabel}
```

Comment:
If the label is a property then this will not work.

Reply:
I see I have caused some confusion here. Cause the general conversation around localization was from my side focusing on the ability to bring arguments to the labels.  Looking at these specific four labels, then I only think it makes sense to accept a method when there is arguments to be considered for the label.

These four does not use any arguments as it stands right now. So I think they can just be of type string and then this is all fine :-)